test/controllers/deenrolment/DeenrolmentProxyControllerSpec.scala

```diff
@@ -28,7 +28,7 @@ class DeenrolmentProxyControllerSpec extends ControllerSpecBase {
 
   "DeenrolmentProxy Controller" must {
 
-    for (enrolment <- Enrolments.values) {
+    for (enrolment <- Enrolments.values.filterNot(_ == Enrolments.AddCis)) {
```

Comment:
This will soon get crowded when we create new pages. Dont have to change now but can if you want.

One idea is to create a list of enrollments above and just remove new ones when we need to, rather than add lots of conditions to this filter as we go on. Might be a better way but its an idea

```scala
val enrolments =
      Enrolments.values - (Enrolments.AddCis, Enrolment, Enrolment)

    for (enrolment <- enrolments) {
       ...    
     }
'''